order/buyer/issue/indicators.md

```diff
@@ -82,3 +91,35 @@ The order or line having logistics status `Open`, `Produced`, `ReadyToShip` or `
 `proposeWhenAccepted`: If this flag is set to true then this line becomes automatically a supplier proposal in case the supplier the accepts a line.
 
 {% page-ref page="propose-when-accepted.md" %}
+
+## Single delivery
+
+When working with the single delivery per order line feature, lines with the same item, prices and terms are grouped into one parent order line with a delivery schedule. In this case, these indicators behave slighty different:
+
+### Confirmed by buyer
+
+Only the parent order line will become confirmed, when all grouped lines are confirmed.
+
+### Reconfirmation request by buyer
+
+Only the parent order line has to be reconfirmed.
+
+### Shipped by supplier
+
+This line will become shipped, and the parent order line will become shipped when all grouped lines are marked as shipped.
```

Comment:
Nit; marked as shipped "Or have status further in the shipping process"?

Reply:
True but this is the essence, which can be easily explained.